flow/Makefile

```diff
@@ -709,4 +709,4 @@ $(foreach file,$(RESULTS_DEF),gui_$(file)): gui_%:
 #-------------------------------------------------------------------------------
 include $(UTILS_DIR)/utils.mk
 export PRIVATE_DIR = ../../private_tool_scripts
--include $(PRIVATE_DIR)/util/utils.mk
+-include $(PRIVATE_DIR)/private.mk
```

Comment:
Do not change this. If you need to include a separate Makefile, make a new line.

Reply:
@rovinski private.mk includes utils.mk (and others).  Why is it better to have two includes here?

Reply:
I don't believe it did at the time of the comment. If it does now, then sure. I prefer having a single private makefile rather than several fragmented makefiles because it makes it easier to find, but that's just me.

Reply:
Modularity. To have a better control whether to add or subtract. Each sub directory represent different tool.

Reply:
Still speaking personally, we've had the modularity debate many times within my group and I don't see the benefit to splitting up rules across Makefiles rather than having a single mono-Makefile. Multiple Makefiles just makes it harder to chase down different targets and trace dependencies between different rules. There is basically no overhead to having rules in a Makefile that are unused, but the readability is greatly enhanced IMO.